lib/sufia/migration/survey/fedora_id_service.rb

```diff
@@ -49,7 +49,12 @@ def active_fedora_model(id)
             query = 'id:"' + id + '"'
             matches = ActiveFedora::SolrService.query(query)
             return nil if matches.count == 0
-            model_str = matches.first["active_fedora_model_ssi"]
+            model_str = matches.first["active_fedora_model_ssi"] || matches.first["has_model_ssim"]
```

Comment:
All version of ActiveFedora (as far back as I can remember) have `has_model_ssim`. The new versions do not use `active_fedora_model_ssi`. I'd prefer that we just change to `has_model_ssim` as it should be completely compatible.